boxes/edges.py

```diff
@@ -1512,10 +1515,21 @@ def __call__(self, l, **kw):
             self.hole(-t, -s-p, p, tabs=4)
             self.rectangularHole(-0.5*t, -s-p-0.5*pinh, t, pinh)
 
-        poly = (0, -180, t, (270, p+s), 0, -90, l+t-p-s)
+        if self.settings.finger_joints_on_box:
+            final_segment = t-s
+            draw_rest_of_edge = lambda : self.edges["F"](l-p)
```

Comment:
I wonder if it is easier to read of you just use a length for the finger joint edge and set that to `0.0` for the normal case. But that's more a question of style.

Reply:
@florianfesti I'll leave this one up to you; this is your code base, and it's important for you to be happy with the style. So if that's how you would prefer, then I'd be happy to make the change.

Reply:
Fair enough. Let's not hold this up on questions of style.